src/object.d

```diff
@@ -728,6 +738,10 @@ class TypeInfo_Function : TypeInfo
         return 0;       // no size for functions
     }
 
+    override const(void)[] init() const @safe
+    {
+        return null;
+    }
     TypeInfo next;
```

Comment:
Nitpick: I like to insert a blank line between member functions and instance fields.